apps/docs/content/pages/getting-started.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: Getting started
+description: Getting started with Workleap Design Tokens
+---
+
+This guide will help you get started with Hopper UI. Hopper UI is a Design System that provides a collection of components, tokens, and icons to help you build applications.
+
+## Quick Install
+
+To use Hopper in your project, run one of these commands in your terminal.
+
+<PackageInstallation library="components @hopper-ui/icons @hopper-ui/tokens @hopper-ui/styled-system" mod="prod" />
+
+After installing Hopper UI packages, you must setup a Provider at the root of your application. This is typically in your _index.tsx_ or _app.tsx_, but can vary based on your specific setup.
+
+```jsx
+import { HopperProvider } from "@hopper-ui/components";
+
+<HopperProvider colorScheme="light">
+    <Button>Hello World</Button>
+</HopperProvider>
+```
+
+<Callout>Hopper is compatible with React 18. Not using React, or using an older version than 18? You can still benefit from Hopper, read on to discover how.</Callout>
+
+## Detailed Installation
+
+### Components
```

Comment:
I've noticed that there is not much difference in style/size between h3-4. I feel like it ends up being a bit messy or hard to follow. Not sure if this would be something easy to fix or not.

Reply:
The main issue is that h5 were so small that I bumped them, I'll see what can be done

Reply:
Here's the current situation: 
h5: 16px
h4: 18px
h3: 20px
h2: 24px
h1: 40px

If h3 goes to 24, h2 would need in theory need to go to 32px its big and close to the h1

![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/fcb6569d-713e-47a1-8603-0f4bee2cd033)

Reply:
We are using the system fonts, therefore we don't have much control over boldness, as we can't assume a system font will support a semi bold.